plugins/shadow-block-converter/test/shadow_block_converter_test.mocha.js

```diff
@@ -11,7 +11,7 @@ const {shadowBlockConversionChangeListener} = require('../src/index');
 
 const assert = chai.assert;
 
-suite('shadowBlockConversionChangeListener', function () {
+suite.skip('shadowBlockConversionChangeListener', function () {
```

Comment:
Should we perhaps file a TODO to re-enable these and add a line comment here pointing to that issue?

Or, do these pass now that the `package-lock.json` file has been corrected?

Reply:
Filed https://github.com/google/blockly-samples/issues/2525